src/main/sarl/fr/utbm/info/vi51/worldswar/environment/PerceptionGrid.java

```diff
@@ -0,0 +1,69 @@
+package fr.utbm.info.vi51.worldswar.environment;
+
+import java.awt.Point;
+import java.util.List;
+
+import javax.naming.directory.InvalidAttributesException;
+
+import fr.utbm.info.vi51.worldswar.environment.envobject.AgentBody;
+import fr.utbm.info.vi51.worldswar.environment.envobject.EnvironmentObject;
+import fr.utbm.info.vi51.worldswar.utils.Grid;
+
+public class PerceptionGrid {
+
+	private final Grid<PerceptionCell> perceptionCellGrid;
+
+	// TODO - c'est dégeu, trouver solution
+	public PerceptionGrid(Grid<PerceptionCell> perceptionCellGrid, Object blblblbJeSertARien) {
```

Comment:
Si tu t'ennuies et que tu veux savoir pourquoi ça marche pas : 
http://stackoverflow.com/questions/1998544/method-has-the-same-erasure-as-another-method-in-type/8467804#8467804

Je pense qu'un workaround un peu plus propre serait de garder le premier constructeur (celui juste au-dessus) en enlevant le Object, et remplacer le deuxième par une méthode statique qui construit la Grid de PerceptionCell, puis appelle le premier constructeur.